pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -117,7 +117,13 @@ def on_train_epoch_start(self, trainer, pl_module):
         self._snap_inter_step_time = None
 
     @rank_zero_only
-    def on_train_batch_start(self, trainer, pl_module, batch, batch_idx, dataloader_idx):
+    def on_train_batch_start(self, trainer, pl_module, *_):
```

Comment:
can we please not start abbreviating stuff like this? This is not good in terms of readability and also one has to spin up a debugger to know what is inside here, if anything needs to be changed.

Reply:
You would still need to evaluate it if it was instead named `*args`. Underscore is just the Python convention for unused arguments.

And using `*args` has the benefit of not having to update the signature here if an argument is added in the future to `on_train_batch_start`.

Reply:
@justusschock should I updated with the whole signature here or `*args and **kwargs` are good as well as @carmocca suggested?